components/github/sources/new-review-request/new-review-request.js

```diff
@@ -1,47 +1,49 @@
-const github = require("../../github.app.js");
+const common = require("../common-polling.js");
```

Comment:
This one can also be replaced by a webhook:
https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#pull_request

Reply:
This one emits an event when you or a team you're a member of are requested to review a pull request. Using a repository webhook would change the functionality to emit an event any time there is a review request opened within a specified repository. We can change it if you think that would be better, but it would alter the way this component works.

Reply:
Yup, you're correct.